src/StatusQ/Popups/StatusSpellcheckingMenuItems.qml

```diff
@@ -0,0 +1,84 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+
+StatusPopupMenu {
+    id: root
+
+    property alias suggestions: suggestionsMenu.model
+
+    signal addToUserDict()
+    signal disableSpellchecking()
+
+    Repeater {
+        id: suggestionsMenu
+
+        delegate: Loader {
+            sourceComponent: root.delegate
+            onLoaded: {
+                item.action.text = modelData
+            }
+        }
+    }
+
+    StatusMenuSeparator {}
+
+    MenuItem {
+        id: ignoreWord
+        implicitWidth: parent ? parent.width : 0
+        implicitHeight: 38
+        contentItem: StatusBaseText {
+            rightPadding: 8
+            leftPadding: 4
+
+            horizontalAlignment: Text.AlignLeft
+            verticalAlignment: Text.AlignVCenter
+
+            text: qsTr("Add to dictionary")
+            color: Theme.palette.directColor1
+            font.pixelSize: 13
+            elide: Text.ElideRight
+        }
+
+        background: Rectangle {
+            color: ignoreWord.hovered ? Theme.palette.statusPopupMenu.hoverBackgroundColor
+                           : "transparent"
+        }
+
+        onTriggered: {
+            root.addToUserDict()
+        }
+    }
```

Comment:
Any reason we're not using `StatusMenuItem` here?

Reply:
StatusMenuItem is an Action, not MenuItem and i dont know why but i have wrong order if use Action with Repeater in menu.